pkg/assets/config/message-with-file-not-found.txt

```diff
@@ -0,0 +1,7 @@
+{{.Thumbnail}}
```

Comment:
could you please name this file as `message-with-hint-file-not-found.txt` to make it clear that it is related to the hint file